src/components/__tests__/FlawForm.spec.ts

```diff
@@ -423,73 +423,6 @@ describe('FlawForm', () => {
     expect(result.owner).toBe('networking test owner');
   });
 
-  it('displays correct Cvss3 calculator link for cvss3 value', async () => {
-    const flaw = sampleFlaw();
-    flaw.cvss3 = '2.2/CVSS:3.1/AV:N/AC:H/PR:H/UI:N/S:U/C:L/I:N/A:N';
-
-    flaw.cvss_scores = [
-      {
-        comment: 'The CVSS is as it is and that is it.',
-        cvss_version: 'V3',
-        flaw: '3dcaf61a-48a7-4483-b1c8-92f56f829abe',
-        issuer: 'RH',
-        score: 2.2,
-        uuid: '23ea1399-219a-4183-ad74-37edd869b2f0',
-        vector: 'CVSS:3.1/AV:N/AC:H/PR:H/UI:N/S:U/C:L/I:N/A:N',
-        embargoed: false,
-        created_dt: '2021-08-02T10:49:35Z',
-        updated_dt: '2024-02-28T14:51:02Z',
-      },
-    ];
-  });
-
-  it('displays correct CVSSv3 calculator link for empty value', async () => {
-    const flaw = sampleFlaw();
-    flaw.cvss_scores = [];
```

Comment:
The test isn't asserting anything, but I'm guessing we want to keep the test so we can assert something later, because cvss_scores wasn't targeted for removal

Reply:
Looks like the expectations/assertions were removed in https://github.com/RedHatProductSecurity/osim/commit/de0c0990161f9731e021e5aadc51ded2becf243c#diff-967a205363234f686e2f6df964ed2631bc6e26c2ba4f9d69abad87898d0d3461L397-L431 @C-Valen wydt?

Reply:
That's right, I asked @C-Valen about it and told me to remove them